openmetadata-docs-v1/content/v0.13.2/connectors/database/pinotdb/airflow.md

```diff
@@ -0,0 +1,736 @@
+---
+title: Run PinotDB Connector using Airflow SDK
+slug: /connectors/database/pinotdb/airflow
+---
+
+# Run PinotDB using the Airflow SDK
+
+{% multiTablesWrapper %}
+
+| Feature            | Status                       |
+| :----------------- | :--------------------------- |
+| Stage              | PROD                         |
+| Metadata           | {% icon iconName="check" /%} |
+| Query Usage        | {% icon iconName="cross" /%} |
+| Data Profiler      | {% icon iconName="check" /%} |
+| Data Quality       | {% icon iconName="check" /%} |
+| Lineage            | Partially via Views          |
+| DBT                | {% icon iconName="cross" /%} |
```

Comment:
@MilanBariya please add the dbt as `check`

Reply:
Done